src/desktop/apps/partner2/test/templates.test.js

```diff
@@ -18,15 +18,24 @@ const render = function (templateName) {
 describe("Partner header", () =>
   describe("canonical links", function () {
     beforeEach(function () {
-      return (this.profile = new Profile(fabricate("profile")))
+      return (this.profile = new Profile(
+        fabricate("profile", {
+          id: "pace-gallery",
+        })
+      ))
     })
 
     it("has a canonical link to the full artist page on partner artist pages", function () {
       this.template = render("index")({
         profile: this.profile,
-        sd: { APP_URL: "http://localhost:3004", CURRENT_PATH: "/pace-gallery" },
+        sd: {
+          APP_URL: "http://localhost:3004",
+        },
         asset() {},
-        params: { id: "pace-gallery", artistId: "yoshitomo-nara" },
+        params: {
+          id: "pace-gallery",
+          artistId: "yoshitomo-nara",
+        },
       })
       return this.template.should.containEql(
```

Comment:
So this validates that the artist canonical is preserved. That was the biggest thing to check so 👍